rtcweb-overview.xml

```diff
@@ -457,12 +457,12 @@
 
       <t><list style="symbols">
           <t>Data transport: TCP, UDP and the means to securely set up
-          connections between entities, as well as the functions for deciding
-          when to send data: Congestion management, bandwidth estimation and
-          so on.</t>
+          connections between entities, as well as the functions for
+          deciding when to send data: congestion management, bandwidth
+          estimation and so on.</t>
 
-          <t>Data framing: RTP, SCTP and other data formats that serve as
-          containers, and their functions for data confidentiality and
+          <t>Data framing: RTP, SCTP, DTLS and other data formats that serve
```

Comment:
Yeah I'll put the comma in.  I'd rather not get embroiled in an oxford comma debate.